process/best_practices_badge.md

```diff
@@ -12,7 +12,73 @@ This document outlines specific questions Academy Software Foundation projects h
+### The project MUST have a publicly available archive for reports and responses for later searching.
+### The project MUST publish the process for reporting vulnerabilities on the project site.
+### If private vulnerability reports are supported, the project MUST include how to send the information in a way that is kept private.
+### The project's initial response time for any vulnerability report received in the last 6 months MUST be less than or equal to 14 days.
+### If the software produced by the project requires building for use, the project MUST provide a working build system that can automatically rebuild the software from source code.
+### It is SUGGESTED that common tools be used for building the software.
+### The project SHOULD be buildable using only FLOSS tools.
+### The project MUST use at least one automated test suite that is publicly released as FLOSS (this test suite may be maintained as a separate FLOSS project). The project MUST clearly show or document how to run the test suite(s) (e.g., via a continuous integration (CI) script or via documentation in files such as BUILD.md, README.md, or CONTRIBUTING.md).
+### A test suite SHOULD be invocable in a standard way for that language.
+### It is SUGGESTED that the test suite cover most (or ideally all) the code branches, input fields, and functionality.
+### It is SUGGESTED that the project implement continuous integration (where new or changed code is frequently integrated into a central code repository and automated tests are run on the result).
+### The project MUST have a general policy (formal or not) that as major new functionality is added to the software produced by the project, tests of that functionality should be added to an automated test suite.
+### The project MUST have evidence that the test_policy for adding tests has been adhered to in the most recent major changes to the software produced by the project."
+### It is SUGGESTED that this policy on adding tests be documented in the instructions for change proposals."
+### The project MUST enable one or more compiler warning flags, a safe language mode or use a separate "linter" tool to look for code quality errors or common simple mistakes if there is at least one FLOSS tool that can implement this criterion in the selected language.
+### The project MUST address warnings.
+### It is SUGGESTED that projects be maximally strict with warnings in the software produced by the project, where practical.
+### The project MUST have at least one primary developer who knows how to design secure software.
 
 A great starting point for all projects is the [Concise Guide for Developing More Secure Software](https://github.com/ossf/wg-best-practices-os-developers/blob/main/docs/Concise-Guide-for-Developing-More-Secure-Software.md) maintained by OpenSSF. Past there, having a primary developer take the three courses outlined below ( all free to audit ) would be sufficient:
```

Comment:
"Show Details" has more info. It doesn't list the concise guide, though, that's a a good idea.

... were you aware of "Show Details"? Let's chat if not!

Reply:
As discussed - lets incorporate all of the aligned feedback into the badge "Show Details". I think in some cases a few minor changes can help.